lib/checkPullRequestBranch.js

```diff
@@ -36,10 +36,11 @@ const checkBranch = async (context) => {
     const commentParams = context.repo({
       issue_number: pullRequest.number,
       body:
-        'Hi @' + prAuthor + ', PRs made from develop, release or test ' +
-        'branches are not allowed. So this PR is being closed. Please make ' +
-        'your changes in another branch and send in the PR. To learn more ' +
-        'about contributing to Oppia, take a look at our ' + wiki + '. Thanks!'
+        'Hi @' + prAuthor + ', PRs made from develop branch or prefixed with ' +
+        'release- or test- are not allowed. So this PR is being closed. ' +
+        'Please make your changes in another branch and send in the PR. To ' +
+        'learn more about contributing to Oppia, take a look at our '
+        + wiki + '. Thanks!'
```

Comment:
Change to:

        'Hi @' + prAuthor + ', PRs made from develop branch or from a branch whose name is prefixed with ' +
        'develop, release or test are not allowed. So this PR is being closed. ' +
        'Please make your changes in another branch and send in the PR. To ' +
        'learn more about contributing to Oppia, take a look at our '
        + wiki + '. (Rule 1 specifically) Thanks!'

Reply:
Done.
Thanks!